bundles/org.openhab.binding.wemo/README.md

```diff
@@ -18,23 +18,47 @@ The binding does not need any configuration.
 
 ## Thing Configuration
 
-For manual Thing configuration, one needs to know the UUID of a certain WeMo device.
-In the thing file, this looks e.g. like
+For manual Thing configuration, one needs to know the UDN of a certain WeMo device. It can most easily be obtained
+by performing an auto-discovery before configuring the thing manually.
 
-```
-wemo:socket:Switch1 [udn="Socket-1_0-221242K11xxxxx"]
-```
+Most devices share the `udn` configuration parameter:
 
-For a WeMo Link bridge and paired LED Lights, please use the following Thing definition
+| Configuration Parameter | Description                        |
+|-------------------------|------------------------------------|
+| udn                     | The UDN identifies the WeMo device |
 
-```
-Bridge wemo:bridge:Bridge-1_0-231445B01006A0 [udn="Bridge-1_0-231445B010xxxx"] {
-MZ100 94103EA2B278xxxx [ deviceID="94103EA2B278xxxx" ]
-MZ100 94103EA2B278xxxx [ deviceID="94103EA2B278xxxx" ]
-}
-```
+### WeMo LED Light
+
+For LED Lights paired to a WeMo Link bridge, please use the following configuration parameter:
+
+| Configuration Parameter | Description                                     |
+|-------------------------|-------------------------------------------------|
+| deviceID                | The device ID identifies one certain WeMo light |
+
+### WeMo Insight Switch
 
+The WeMo Insight Switch has some additional parameters for controlling the behavior for channel `currentPower`.
+This channel reports the current power consumption in Watt. The internal theoretical accuracy is 5 mW, i.e.
```

Comment:
Please cut, one sentence per line is the recommendation.
Same remark for all the file (and all your changes).

Reply:
Done for whole file.